x-pack/plugins/siem/public/lib/connectors/components/connector_flyout/index.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useEffect } from 'react';
+import { EuiFieldText, EuiFlexGroup, EuiFlexItem, EuiFormRow, EuiSpacer } from '@elastic/eui';
+
+import { isEmpty, get } from 'lodash/fp';
+
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { ActionConnectorFieldsProps } from '../../../../../../triggers_actions_ui/public/types';
+import { FieldMapping } from '../../../../pages/case/components/configure_cases/field_mapping';
+
+import { defaultMapping } from '../../config';
+import { CasesConfigurationMapping } from '../../../../containers/case/configure/types';
+
+import * as i18n from '../../translations';
+import { ActionConnector, ConnectorFlyoutHOCProps } from '../../types';
+
+export const withConnectorFlyout = <T extends ActionConnector>({
```

Comment:
Really nice refactor exactly what we wanted <3

Reply:
Thank you!